app/jobs/attach_files_to_file_set_job.rb

```diff
@@ -0,0 +1,75 @@
+# Converts UploadedFiles into FileSets and attaches them to works.
+class AttachFilesToFileSetJob < AttachFilesToWorkJob
```

Comment:
I think we need to extend the functionality to accept the file use parameter and perform ingest files with those files use properties provided as well.